README.md

```diff
@@ -119,6 +119,49 @@ Connection: keep-alive
 
 This error shows the request is missing a required property `name` from the HTTP request body.
 
+### HTTP API
+
+Prism provides [HTTP API][http-docs] as separate module for forwarding and mocking requests.
+
+#### Forwarding example
```

Comment:
We’re trying to hide forwarding until we can figure out how we want to do it properly, not advertise it. Could we remove this one?

Reply:
I will look into it and reword it.